client/src/components/ResearchArea/ReseachArea.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Soldier from './ResearchAreaImg/Group-2491.svg';
+import Runner from './ResearchAreaImg/Group-2492.svg'
```

Comment:
I am not sure this should be in components. This is part of Home Page. I would put it in client/src/pages/Home/